packages/discord.js/src/structures/ChannelSelectMenuInteraction.js

```diff
@@ -11,6 +11,12 @@ class ChannelSelectMenuInteraction extends MessageComponentInteraction {
   constructor(client, data) {
     super(client, data);
 
+    /**
+     * Array of selected channels' ids
```

Comment:
Ditto, this also feels jank, thoughts on how to better phrase those? @Jiralite

Reply:
> An array of the selected channel ids

How about this? I'm not really sure myself.

Reply:
may be